scalalib/src/mill/scalalib/publish/SonatypeHelpers.scala

```diff
@@ -0,0 +1,75 @@
+package mill.scalalib.publish
+
+import mill.util.Jvm
+
+import java.math.BigInteger
+import java.security.MessageDigest
+
+trait SonatypeHelpers {
```

Comment:
Looks like there are only helper `def`s inside which don't use any state and don't seem to be customized via inhertance, so we could as well make it an `object` and avoid the `extends` on the use site.

Reply:
Done.